src/features/projects/screens/Projects.tsx

```diff
@@ -164,15 +164,6 @@ function ProjectsList({
         <div
           style={{ transform: `translate(0,${scrollY}px)` }}
```

Comment:
I assume the whole state `scrollY` is not used any more now and could be removed everywhere?